staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2982,6 +2982,58 @@ type PodSpec struct {
 	// Optional: Defaults to true.
 	// +optional
 	EnableServiceLinks *bool `json:"enableServiceLinks,omitempty" protobuf:"varint,30,opt,name=enableServiceLinks"`
+	// TopologySpreadConstraints describes how a group of pods ought to spread across topology
+	// domains. Scheduler respects and enforces the constraints.
```

Comment:
Change the last sentence to "Scheduler respects the constraints during scheduling."

Does another component work to make the constraints true after pods have been scheduled (i.e. by evicting pods)?

Reply:
At this moment, no. We don't proactively balance the pods by evicting pods.